app/assets/stylesheets/results.scss

```diff
@@ -4,4 +4,9 @@ tr.solid-top-border {
 
 tr.solid-bottom-border {
   border-bottom: 2px solid #000
-}
\ No newline at end of file
+}
+
+// make the confirmation panel a neutral colour
+.govuk-panel--confirmation {
```

Comment:
We _also_ use a `.govuk-panel--confirmation` on /referrals. We currently override it to be grey background, but this new CSS rule takes precedence and turns it blue!

I recommend adding a new class for blue panels that we can apply to the results screen to ensure only the intended panel gets changed